shell/platform/common/cpp/client_wrapper/core_implementations.cc

```diff
@@ -33,8 +33,7 @@ namespace {
 // interface provided by the C API and the std::function-based message handler
 // interface of BinaryMessenger.
 void ForwardToHandler(FlutterDesktopMessengerRef messenger,
-                      const FlutterDesktopMessage* message,
-                      void* user_data) {
+                      const FlutterDesktopMessage* message, void* user_data) {
```

Comment:
If possible, can you not make change like this at original part. In my case, it interferes with reading the code.

Reply:
oK,I will revert to the original。